docs/src/standalone/pages/vegetation/radiative_transfer/beer_model.md

```diff
@@ -0,0 +1,59 @@
+# Radiative transfer scheme
+This section describes multiple models of radiative transfer 
+through the vegetation canopy, implemented in ClimaLSM. 
+
+## Beer's law
+Plants utilize Photosynthetically Active Radiation (PAR) for the process of photosynthesis, during which they convert light energy into chemical energy, fueling the synthesis of sugars and other organic compounds. PAR refers to the portion of the electromagnetic spectrum that is essential for photosynthesis in plants. PAR includes wavelengths ranging from approximately 400 to 700 nanometers and corresponds to the visible light spectrum. The unit used to measure PAR is called micromoles per square meter per second (μmol/m²/s), representing the number of photons within the PAR range that strike a square meter of a surface per second.
+
+The portion of PAR that is actually absorbed by the vegetation canopy for photosynthesis is called Absorbed Photosynthetically Active Radiation (APAR). The APAR driving photosynthesis is calculated following the Beer-
+Lambert law:
+
+```math
+APAR(PAR, \theta_s) = (PAR)(1 - \rho_{leaf})(1 - e^{(-K(\theta_s) LAI  \Omega)})
+```
+
+where PAR ≈ SW/2 is the incident moles of photons per meter squared per
+second in the PAR window, approximated as half of the incident shortwave flux.
+If PAR is not directly available, $ρ_{leaf}$ is the PAR canopy reflectance, K is the
+vegetation extinction coefficient following Campbell (1998), LAI is the leaf area
+index, $θ_s$ is the zenith angle, and $Ω$ is the clumping index following Braghiere
+(2021). $K$, $Ω$ and $ρ_{leaf}$ are all unitless. LAI is in m² m⁻².
+In order to compute $K$, we need $θ_s$ in radians and the leaf angle distribution $l_d$
+(unitless). K is then defined as
+
+```math
+K = l_d/\max{(\cos{(\theta_s)}, \epsilon)}
+```
+
+so that at night, when 3π/2 > $θ_s$ > π/2, $K$ is large (lots of extinction) and
```

Comment:
Perhaps instead of "lots of extinction" we could say something like very little light penetrates through the canopy? Since it's not that more incident photons "become extinct" but rather that there are just very few incident photons at night.